pagopa/io-app
# chore: [IOBP-704] Add payed rptid into state entities on payment success

## Short description
This PR adds the rptId when a payment is successfully made within the state.entities to display the "Paid" badge in the message list.

## List of changes proposed in this pull request
- Added an additional orchestration action `paymentCompletedSuccess` to the new payment flow checkout;
- Dispatched the above action when showing the successful outcome;
- Added the action handler into `ts/store/reducers/entities/payments.ts` file

## How to test
- Enable the new wallet landing page FF;
- Start a payment from the new payment landing;
- When you reach the success outcome, you should be able to see the dispatched action with the rptId paid.

## Preview
https://github.com/pagopa/io-app/assets/34343582/ebf23085-8549-49b1-9987-5c7de3a04c75
